google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/flexible/FlexibleFacetEditor.form

```diff
@@ -65,7 +65,7 @@
               <grid row="2" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="1" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
             </constraints>
             <properties>
-              <text value="Generate..."/>
+              <text value="Generate Dockerfile..."/>
```

Comment:
I know it was hardcoded here before..but this should also be in the message bundle.